lib/individual_util.rb

```diff
@@ -53,12 +53,18 @@ def enroll_waived_or_terminated enrollment, status
     enrollment[:plan_name] = 'KP DC Silver 2000/35'
     enrollment[:plan_type] = 'HMO'
     enrollment[:metal_level] = 'Silver'
+    enrollment[:elected_aptc_pct] = 0
+    enrollment[:applied_aptc_amount_in_cents] = 0
 
     enrollment[:carrier] = {
         name: 'Kaiser',
         summary_of_benefits_url: '/document/download/dchbx-enroll-sbc-preprod/ad954b2b-81ca-4729-b440-811eead43498?content_type=application/pdf&filename=UHCChoicePlusHSAPOSGold1300A.pdf&disposition=inline'
     }
 
+    enrollment[:provider_directory_url] = 'http://mydoctor.kaiserpermanente.org/mas/mdo/?kp_shortcut_referrer=kp.org/doctor'
+    enrollment[:rx_formulary_url] = 'https://healthy.kaiserpermanente.org/static/health/pdfs/formulary/mid/mid_exchange_formulary.pdf'
+    enrollment[:services_rates_url] = BaseUtil::url "#{$GENERATED_DIR}/#{ServiceUtil::SERVICES_RATE_JSON}"
+
```

Comment:
Yep, looks like you duplicated this in your commit: 3699a645f09582dffd51bdcdae7c481b68a3d42f.

I'll refactor it.